packages/lesswrong/server/scripts/lessOnlineMessaging.tsx

```diff
@@ -0,0 +1,142 @@
+    </p>
+    <hr/>
+    <p>
+      P.S. This festival started as an idea when we were talking to the team behind the forecasting & prediction market festival <b><a href="https://www.manifest.is/">Manifest</a></b>. They ran a smash-hit event last year at Lighthaven, and this year they encouraged us to run our own event as part of a series. This means that the weekend right after LessOnline is Manifest, and we've <b><a href="https://less.online/#tickets-section">teamed up to offer a discounted ticket to both</a></b>. Already many people have made plans to come for both and will be staying at Lighthaven for the week in between!
+    </p>
+    <p>
+      P.P.S. <b>Got questions?</b> You can’t reply to this email, but <b>comments and questions are welcome in <a href="https://www.lesswrong.com/posts/MmWziepD8DDauSide/lessonline-festival-updates-thread">this LW thread</a></b> and/or using <b>the intercom bubble</b> in the bottom right of the <a href="https://less.online/">less.online</a> or <a href="http://lesswrong.com">lesswrong.com</a> websites.
+
+    </p>
+    <img src="https://res.cloudinary.com/lesswrong-2-0/image/upload/v1712166847/habryka_6_g84ubl.png"/>
+  </div>
+}
+
+const lessOnlineMessaging = async () => {
+  // const lwevents = await LWEvents.find({createdAt: {$gte: new Date("2024-01-28")}}).fetch()
+  // console.log(lwevents.length)
+
+  // console.log(lwevents.length, Array.from(new Set(lwevents.map(event => event.userId))).length)
+
+  // const users = await Users.find({
+  //   lastNotificationsCheck: {$gt: new Date("2020-01-01")},
+  //   $or: [{ banned: { $exists: false } }, { banned: { $lte: new Date() } }],
+  //   karma: {$gte: 0}
+  // }).fetch()
+
+  // const ignoredDisplayNames = ['Elizabeth', 'Scott Alexander', 'Eliezer Yudkowsky', 'Tsvi Benson-Tilsen', 'TsviBT', '[DEACTIVATED] Duncan Sabien', 'Raelifin', 'PeterMcCluskey', 'Peter_McCluskey', 'ozymandias', 'Rob Bensinger', 'Ziz', 'Gwen_', 'ialdabaoth', 'Holly_Elmore', 'David_Gerard', 'sarahconstantin', 'Sarah Constantin', 'LoganStrohl', 'Malcolm Ocean', 'MalcolmOcean', 'johnswentworth', 'eukaryote', 'Eric Neyman', 'Eneasz Brodski', 'Eli Tyre', 'Malmesbury']
+
+  // const usersMinusLessOnline = users.filter(user => ignoredDisplayNames.includes(user.displayName))
+  // console.log(users.length)
+
+  const raemon = await Users.findOne({username: "Raemon"})
+  const benito = await Users.findOne({displayName: "Ben Pace"})
+  const habryka = await Users.findOne({displayName: "habryka"})
+  const kave = await Users.findOne({displayName: "kave"})
+  const robert = await Users.findOne({displayName: "RobertM"})
+  const users = [
+    raemon, 
+    // benito, 
+    robert
+    // habryka, 
+    // kave
+  ]
+  
+  for (const user of users) {
+    if (user) {
+      await wrapAndSendEmail({
+        user: user,
+        subject: "LessOnline: A Festival of Truth-Seeking and Blogging (May 31 — Jun 2, Berkeley CA)",
+        body: LessOnlineEmail()
```

Comment:
I don't think there's any reason to make this a function if it's always returning the same thing